libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/RouteHighlightOverlay.kt

```diff
@@ -96,9 +105,19 @@ class RouteHighlightOverlay(context: Context, private val mapView: CycleMapView)
 
         routeNowIcon.visibility = View.INVISIBLE
 
+        // Checking if segment is a Start saves having to define dummy summaryText methods in the other Segment classes
```

Comment:
This comment makes me think that I should see changes to "the other Segment classes" in this PR, removing the dummy methods you appear to be referring to.  Have I misunderstood?

Reply:
summaryText is a new method, only required in the Start segment.  I would have had to add it to the other Segment  classes as a dummy (empty) method, but instead I check here whether the Segment is a Start segment, to save doing that.
Should I remove that comment?